dadguide/monster_graph.py

```diff
@@ -118,12 +117,13 @@
 
 
 class MonsterGraph(object):
-    def __init__(self, database: DadguideDatabase):
+    def __init__(self, database: DadguideDatabase, debug_mons: Optional[List[int]] = None):
```

Comment:
did you really measure perf of this and are we sure we need to do anything here?